components/list/list.js

```diff
@@ -18,7 +24,11 @@ function List(props) {
 
   const className = getClassNamesWithMods('ui-list', mods);
 
-  const itemsBlock = items.map((item, index) => (
+  const notEmpty = i => i !== '' && i !== null;
+
+  const itemsToRender = (renderEmptyItems === false) ? items.filter(notEmpty) : items;
```

Comment:
Good one. You could've also filter by `Boolean`:

```js
items.filter(Boolean)
```

Reply:
But it that case we will get 0 values filtered out, I wanted to prevent that. @mAiNiNfEcTiOn